drivers/wifi/esp_at/Kconfig.esp_at

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2019 Tobias Svehagen
 # SPDX-License-Identifier: Apache-2.0
 
-menuconfig WIFI_ESP
+menuconfig WIFI_ESP_AT
 	bool "Espressif ESP8266 and ESP32 support"
```

Comment:
Would be good to update the string here / add a help test to explain this is for `AT`.

Reply:
Updated as requested.